gradio/chat_interface.py

```diff
@@ -205,6 +206,7 @@ def __init__(
                         example_message["icon"] = example_icons[index]
                     examples_messages.append(example_message)
 
+            self.provided_chatbot = chatbot is not None
```

Comment:
seems a bit unnecessary to create this variable

Reply:
It's necessary to check later on if the user provided a chatbot in render. If the user provided no chatbot, then we reset the height later to 400 if rendered inside another Blocks. But if a user specified a chatbot, then they may have set a specific height as well, and we don't want to overwrite that.